### PR TITLE
pytorch ac_simple_navigation

### PR DESCRIPTION
**Pull request by emailweixu**
Also generated more timing summary for on_policy_algorithm

### CODE REVIEW

File: alf/algorithms/on_policy_algorithm.py [hunk 1]
```diff
@@ -66,14 +67,18 @@ def train_step(self, exp: Experience, state):
...
+
+        with record_time("time/train"):
+            training_info = training_info._replace(
+                rollout_info=(), info=training_info.rollout_info)
+            valid_masks = (training_info.step_type != StepType.LAST).to(
+                torch.float32)
+            loss_info, params = self.update_with_gradient(
+                self.calc_loss(training_info), valid_masks)
+            self.after_update(training_info)
+            self.summarize_train(training_info, loss_info, params)
```
Comment (le-horizon) [inline alf/algorithms/on_policy_algorithm.py:82]: while you are at it, let's also separately log summary time?

**Review by le-horizon — APPROVED** [review]
LG.  Just curious, does TracAlgo improve training stability here?

File: alf/algorithms/on_policy_algorithm.py (hunk 1 above)
Comment (emailweixu) [reply alf/algorithms/on_policy_algorithm.py:82]: summary is distributed in lot of places. It's not easy to collect them all.

**Comment by emailweixu** [comment]
> LG. Just curious, does TracAlgo improve training stability here?

No clear benefit.

**Pull request merged**